#!/usr/bin/env python
PACKAGE = "ros_assignment"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("H_min", int_t, 0, "Lower bound for H value", 70, 0, 179)
gen.add("H_max", int_t, 0, "Upper bound for H value", 130, 0, 179)
gen.add("S_min", int_t, 0, "Lower bound for s value", 10, 0, 255)
gen.add("S_max", int_t, 0, "Upper bound for s value", 255, 0, 255)
gen.add("V_min", int_t, 0, "Lower bound for v value", 10, 0, 255)
gen.add("V_max", int_t, 0, "Upper bound for v value", 255, 0, 255)

exit(gen.generate(PACKAGE, "hsv", "Hsv"))
